### ISSUE TITLE
Resizing panes in some editors 

### ISSUE THREAD

**Issue by Tubbles** [LibrePCB/LibrePCB#815]
##### SUMMARY

In some editors, the compartments where the view panes are placed are not resizable, making them take up a lot of dead space on high resolution monitors.

Specifically I would expect the following pane delimiters (marked red) be resizable:

Device Editor
![image](https://user-images.githubusercontent.com/2370561/103177352-f2ff6a80-4879-11eb-9422-4a396b4a965d.png)

Symbol Editor
![image](https://user-images.githubusercontent.com/2370561/103177384-54273e00-487a-11eb-97f2-2167db2bda41.png)

Package Editor
![image](https://user-images.githubusercontent.com/2370561/103177398-89cc2700-487a-11eb-9bfd-08fb92d709e9.png)

Component Editor
![image](https://user-images.githubusercontent.com/2370561/103177408-9e102400-487a-11eb-849d-936d63e8cff6.png)

**Comment by EdizonTN** [comment]
I agree!

I wrote by the another words here: https://github.com/LibrePCB/LibrePCB/issues/726

**Comment by dbrgn** [comment]
I'd say #726 is a superset of this issue (but they both sound like they might be useful).

This issue could probably be achieved just by fiddling with the layout files.

**Comment by Tubbles** [comment]
> I wrote by the another words here: #726

Argh! Was certain I had searched through the backlog, good stuff! :)

> I'd say #726 is a superset of this issue (but they both sound like they might be useful).
> 
> This issue could probably be achieved just by fiddling with the layout files.

It sounds like this issue is superseded by #726, I propose we close this

**Comment by dbrgn** [comment]
Well, if you just want resizing then keep it open. #726 would be quite a big refactoring that would require touching most UI code and integrating a new library, while this issue can probably be done just by tweaking UI files. So I still see value in a separate feature request with reduced scope.

**Comment by Tubbles** [comment]
Alright, lets keep it then :) I'm not so well versed in Qt but might give it a shot

**Comment by BuFran** [comment]
Yeah I searched for that, but I didn't found this issue. I'm sorry for #1244.  

I can make it because it is only simple touching of *.ui files. 

So i'm just asking @ubruhin , it's nice and easy work for my next PR ? :-)

**Comment by ubruhin** [comment]
The new UI (see #1494) introduces a very new layout of the editors, and increases the working area in several ways. Therefore maybe this issue is not relevant anymore. Let's first gain some experience with the new UI of LibrePCB 2.0, and if there is still a need to resize panels or so, let's address it in a new issue.
